test/api/stream_test.exs

```diff
@@ -58,46 +43,29 @@ defmodule Api.StreamTest do
                |> Enum.take(1)
     end
 
-    # We're seeing occasional failures in this test due to an underlying issue
-    # with the `Bypass` library. There is an
-    # [open issue on Bypass's github](https://github.com/PSPDFKit-labs/bypass/issues/120).
-    test "handles api events", %{bypass: bypass} do
-      Bypass.expect(bypass, fn conn ->
-        conn = Conn.send_chunked(conn, 200)
-
-        data = %{
-          "attributes" => [],
-          "type" => "vehicle",
-          "id" => "vehicle"
-        }
-
-        {:ok, conn} = Conn.chunk(conn, "event: ignores unexpected events\n\n")
-        {:ok, conn} = Conn.chunk(conn, "ignored garbled data\n\n")
-        {:ok, conn} = Conn.chunk(conn, "event: reset\ndata: #{Jason.encode!([data])}\n\n")
-        {:ok, conn} = Conn.chunk(conn, "event: add\ndata: #{Jason.encode!(data)}\n\n")
```

Comment:
Could this introduce flakiness if `Jason.encode!` doesn't follow the same key order each time?

Reply:
I don't believe so, because we're asserting on the type returned and the event returned, but it doesn't seem that we're asserting on any of the JSON data.